src/sentry/sentry_apps/api/serializers/servicehookproject.py

```diff
@@ -0,0 +1,11 @@
+from sentry.api.serializers import Serializer, register
+from sentry.sentry_apps.models.servicehook import ServiceHookProject
+
+
+@register(ServiceHookProject)
+class ServiceHookProjectSerializer(Serializer):
+    def serialize(self, obj, attrs, user, **kwargs):
+        return {
+            "id": obj.id,
+            "project_id": obj.project_id,
```

Comment:
```suggestion
            "id": str(obj.id),
            "project_id": str(obj.project_id),
```

Generally ids are stringified in our APIs as we use big ints and project ids can overflow Javascript Number type.

Reply:
Does this apply too for request serializers :bufo-big-eyes: ? (i.e if a user sends a req to us with an int type project id) could that also overflow?

Reply:
done

Reply:
> Does this apply too for request serializers :bufo-big-eyes: ? (i.e if a user sends a req to us with an int type project id) could that also overflow?

No, python is fine with big numbers.